### PR TITLE
Correcting HasEverPlayed metric entry

### PR DESCRIPTION
**Pull request by sideb0ard**
Cobalt.Media.HasEverPlayed was correctly being logged as base::UmaHistogramBoolean()
however, it was incorrectly using an enum for PipelineStatus.

This CL adds a new enum BooleanHasPlayed and sets Cobalt.Media.HasEverPlayed to use it.
( I used a new enum rather than just the Boolean enum to follow Chromium's example:
https://source.chromium.org/chromium/chromium/src/+/main:tools/metrics/histograms/metadata/media/histograms.xml;l=3891)

b/337308115

### CODE REVIEW

File: tools/metrics/histograms/metadata/cobalt/histograms.xml [hunk 1]
```diff
@@ -119,15 +119,14 @@ Always run the pretty print utility on this file after editing:
   </summary>
 </histogram>
 
-<histogram name="Cobalt.Media.HasEverPlayed" enum="PipelineStatus"
+<histogram name="Cobalt.Media.HasEverPlayed" enum="BooleanHasPlayed"
```
Comment (joeltine) [inline tools/metrics/histograms/metadata/cobalt/histograms.xml:122]: I _think_ changing the enum definition on an existing histogram is mostly safe... but one issue will be that historical data of Cobalt.Media.HasEverPlayed will always use the old PipelineStatus labels. There will be a period time where both sets of labels show up in the dashboard (you'll see 4 unique values), but it should fix itself after the pipeline runs for a week or so. But if you ever query for older dates, you'll see these old labels.

File: tools/metrics/histograms/metadata/cobalt/enums.xml [hunk 2]
```diff
@@ -37,6 +37,13 @@ https://chromium.googlesource.com/chromium/src.git/+/HEAD/tools/metrics/histogra
   <int value="1" label="True"/>
 </enum>
 
+
```
Comment (joeltine) [inline tools/metrics/histograms/metadata/cobalt/enums.xml:40]: Nit: kill a new line before and ever this enum stanza

**Review by joeltine — APPROVED** [review]

File: tools/metrics/histograms/metadata/cobalt/histograms.xml (hunk 1 above)
Comment (sideb0ard) [reply tools/metrics/histograms/metadata/cobalt/histograms.xml:122]: ack!

File: tools/metrics/histograms/metadata/cobalt/enums.xml (hunk 2 above)
Comment (sideb0ard) [reply tools/metrics/histograms/metadata/cobalt/enums.xml]: done

**Pull request merged**